app/util/data_preparation/api/jira_clients.py

```diff
@@ -151,3 +151,37 @@ def get_server_info(self):
         response = self.get(api_url, 'Could not get the server information')
 
         return response.json()
+
+    def get_nodes_count_rest(self):
```

Comment:
This method returns nodes info dict, so better name for this method is `get_nodes_info_via_rest `